### DISCUSSION TITLE
Github Stats

### DISCUSSION THREAD

**Discussion by simpankode** [community/community#19586] [Other Feature Feedback, Questions, & Ideas] [answered]
how to create a GitHub README stats

**Comment by melvincwng** [comment]
@simpankode, follow the instructions in this [code repository](https://github.com/anuraghazra/github-readme-stats) to create a GitHub stats badge for your README.md

Hope this clarifies.

**Reply by melvincwng** [reply]
@simpankode, Thanks, and have a great day ahead 😄.

**Comment by prayogaea** [comment, marked as answer]
visit [https://github.com/anuraghazra/github-readme-stats](https://github.com/anuraghazra/github-readme-stats) and [https://github-readme-stats.vercel.app/](https://github-readme-stats.vercel.app/)
